src/bin/pgcopydb/pgcmd.c

```diff
@@ -829,6 +880,9 @@ pg_restore_db(PostgresPaths *pgPaths,
 		args[argsIndex++] = (char *) listFilename;
 	}
 
+	/* verbose output */
+	//args[argsIndex++] = "--verbose";
+
```

Comment:
I suspect this is meant to be removed later, before final review and merge? You might also be able to use `log_get_level` to selectively add `--verbose` to the pg_dump call when using --debug or --trace at the pgcopydb level?